### PR TITLE
Add Keplr wallet support

### PR DESCRIPTION
**Pull request by williamchong**
Adds Keplr(cosmos wallet) register/login support, implement temp UI for new design

### CODE REVIEW

File: util/Keplr.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+      if (!res) throw new Error('CANNOT_INIT_KEPLR');
+    }
+    const message = {
+      chain_id: network.id,
+      memo: signPayload,
+      msgs: [],
+      fee: { gas: '1', amount: { denom: 'nanolike', amount: '0' } },
+    };
+    const payload = await this.signer.sign(
+      this.accounts[0].address,
```
Comment (nwingt) [inline util/Keplr.js:118]: can replace with `this.getWalletAddress()`?

File: util/User.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import EthHelper from '@/util/EthHelper';
 import FileHelper from '@/util/FileHelper';
-import {
-  LOGIN_MESSAGE,
-} from '@/constant';
+import Keplr from '@/util/Keplr';
+import { LOGIN_MESSAGE } from '@/constant';
+import stringify from 'fast-json-stable-stringify';
```
Comment (nwingt) [inline util/User.js:5]: relative imports should move to bottom

File: util/Keplr.js (hunk 1 above)
Comment (williamchong) [reply util/Keplr.js]: Added a new method for getting address internally

File: components/dialogs/AuthDialog.vue [hunk 3]
```diff
@@ -153,7 +153,10 @@
                     stroke-width="2"
                   />
                 </svg>
-                <div>{{ $t('WalletNoticeDialog.page[1].description[0]') }}</div>
+                <div>
+                  <p>{{ $t('WalletNoticeDialog.page[1].description[0]') }}</p>
+                  <p>{{ $t('WalletNoticeDialog.page[1].description[1]') }}</p>
```
Comment (nwingt) [inline components/dialogs/AuthDialog.vue:158]: Remove the trailing margin from `<p>`
```suggestion
                  <p class="lc-margin-bottom-0">
                    {{ $t('WalletNoticeDialog.page[1].description[1]') }}
                  </p>
```

**Review by nwingt — CHANGES_REQUESTED** [review]

**Review by nwingt — APPROVED** [review]

**Pull request merged**